client/src/main/java/model/FleetCreator.java

```diff
@@ -0,0 +1,59 @@
+package model;
+
+import gui.fields.Mast;
+
+import java.util.ArrayList;
+import java.util.List;
+/**
+ * It holds information regarding a player's fleet.
+ *
+ * @author Patrycja Mikulska
+ * @version 1.5
+ */
+
+public class FleetCreator {
+
+  private List<Position> positions;
+  private final Sea sea;
+  private final Fleet ships;
+  private Ship shipThatIsBuild;
+
+  public FleetCreator(Sea sea) {
```

Comment:
FleetCreator is a complicated class and I can't understand what is happening here.
It does too many things. Its state is not predictable and error-prone. 
If possible, split responsibilities between smaller components.